fastify3/view/index.ejs

```diff
@@ -0,0 +1,20 @@
+<html>
+    <head>
+        <% include partials/head %>
+    </head>
+    <body>
+        <% include partials/header %>
+        <div class="container-fluid">
+            <div class="row">
+                <% include partials/nav %>
+                <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+                    <h1 class="page-header">Contrast Fastify 3 Test Bench</h1>
```

Comment:
I included the 3 ejs files that specify the name of the app. I know we don't _need_ that but I found it helpful.